Product/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+class Main_category(models.Model):
+	main_category = models.CharField(max_length=45)
+
```

Comment:
```suggestion
    class Meta:
        db_table = 'main_categories'
```

테이블 이름 정확하게 명시해주세요 :) 아래 모든 테이블에 적용해주세요